app/components/MigrationDataOverview.tsx

```diff
@@ -1,21 +1,25 @@
 import { type Translations } from "~/services/cms/index.server";
 import { getTranslationByKey } from "~/util/getTranslationByKey";
 import { lookupOrKey } from "~/util/lookupOrKey";
-import Background from "./Background";
-import Box from "./Box";
-import Container from "./Container";
+import Heading from "./Heading";
 
 type MigrationDataProps = {
   readonly migrationData?: Record<string, unknown>;
   readonly translations: Translations;
 };
 
-const getMigrationValueTranslation = (
+const renderMigrationValue = (
   translations: Translations,
-  value: string,
+  value: unknown,
   key: string,
 ) => {
-  const translation = translations[value];
+  if (typeof value === "object" && value !== null) {
+    return Object.entries(value).map(([_, subValue]) => (
+      <p key={subValue}>{lookupOrKey(subValue as string, translations)} </p>
```

Comment:
is this by accident with extra whitespace?
```suggestion
      <p key={subValue}>{lookupOrKey(subValue as string, translations)}</p>
```